components/Events.js

```diff
@@ -14,6 +23,15 @@ export const Events = [
 		end: new Date(2022, 9, 7, 18, 0, 0),
 		description: "REALLY COOL DESCRIPTION THAT WE NEED TO FIGURE OUT",
 	},
+	{
+		title: "Metaverse With Meta: Meta Panel",
+		location: "WCH 205/206",
+		color: filters["Meetings"],
+		start: new Date(2022, 9, 11, 16, 0, 0),
+		end: new Date(2022, 9, 11, 18, 0, 0),
+		description:
+			"The metaverse is a concept about which we know extremely little. Is it the future of the internet? Or a video game? Or a deeply uncomfortable version of Zoom? Join us for our second GM, as we’ll be holding a panel with Meta software engineers, discussing what is the metaverse, how social interaction will change, and their experience working under a MANGA company.",
```

Comment:
Explore the future of the metaverse with Meta software engineers and learn about its impacts in social interaction.